langkit/templates/properties/def_ada.mako

```diff
@@ -29,6 +29,10 @@ is
    Self : ${Self.type.name} := ${Self.type.name}
      (${property.self_arg_name});
 
+   ## Dispatchers must not memoize: it is the job of the static properties do
```

Comment:
"Dispatchers must not memoize because it will be done at the static property level, so we don't want to do it twice" seems clearer/more accurate.